apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -783,6 +814,8 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
         }
 
         // If the vote is still open, it cannot be executed
+        // Note: I think this does not correctly check for the quiet ending flip (and thereby extension)
+        //       If the vote was flipped but not extended through another vote, this will continually stay false, locking the vote from being executed
```

Comment:
The scenario:

1. Vote gets flipped in the last round
2. Time goes by and nobody votes during the extra extension (which is also "missing" / not registered in state)
3. The vote can't be executed, as `_hasEnded()` will continue to stay false, despite the "phantom" extension having been quiet.

Reply:
Addressed in f7a96d8b837a4e03b934a5115638b2a411db2aad